app/app.go

```diff
@@ -1224,21 +1229,9 @@ func (app *Evmos) setupUpgradeHandlers() {
 	switch upgradeInfo.Name {
 	case v19.UpgradeName:
 		// revenue module is deprecated in v19
-		// This ran only on testnet
 		storeUpgrades = &storetypes.StoreUpgrades{
 			Deleted: []string{"revenue"},
-		}
-	case v191.UpgradeName:
-		if utils.IsMainnet(app.ChainID()) {
-			// revenue module is deprecated in v19.1
-			storeUpgrades = &storetypes.StoreUpgrades{
-				Deleted: []string{"revenue"},
-				Added:   []string{ratelimittypes.ModuleName},
-			}
-		} else {
-			storeUpgrades = &storetypes.StoreUpgrades{
-				Added: []string{ratelimittypes.ModuleName},
-			}
+			Added:   []string{ratelimittypes.ModuleName},
 		}
```

Comment:
Nit: if we're removing the v19.1 logic, please also remove the corresponding folder (`app/upgrades/v19_1)`